crypto/codec/amino.go

```diff
@@ -26,20 +27,25 @@ func RegisterCrypto(cdc *codec.LegacyAmino) {
 	// first line.
 	cdc.RegisterInterface((*crypto.PubKey)(nil), nil)
 	cdc.RegisterInterface((*cryptotypes.PubKey)(nil), nil)
-	cdc.RegisterConcrete(ed25519.PubKey{},
-		ed25519.PubKeyName, nil)
 	cdc.RegisterConcrete(sr25519.PubKey{},
 		sr25519.PubKeyName, nil)
+	// TODO Same as above, for ED25519
```

Comment:
Do we have an issue for that, so we can link this code?

Reply:
Not sure if we have that as this would require a large refacto because tendermint pubkeys are still used quite a lot in calling methods from tendermint (one of the reason we do this https://github.com/cosmos/cosmos-sdk/pull/7268#discussion_r493604779), eg https://github.com/tendermint/tendermint/blob/e8eef856196697c803ac54fa625d83fb276d5977/crypto/encoding/codec.go#L12 called in `x/staking/types/validator.go`
I believe that might require a larger architecture decision.

Reply:
actually this could be covered by https://github.com/cosmos/cosmos-sdk/issues/7357 @robert-zaremba